jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -94,14 +94,15 @@ private static final class TidLock {
 	 */
 	private volatile boolean deadInterrupt;
 	/*[ENDIF] JAVA_SPEC_VERSION >= 14 */
-	private volatile boolean started;				// If !isAlive(), tells if Thread died already or hasn't even started
-	private String name;						// The Thread's name
-	private int priority = NORM_PRIORITY;			// The Thread's current priority
+	private volatile boolean started;		// If !isAlive(), tells if Thread died already or hasn't even started
+	private String name;					// The Thread's name
+	private int priority = NORM_PRIORITY;	// The Thread's current priority
 	private boolean isDaemon;				// Tells if the Thread is a daemon thread or not.
+	private volatile int threadStatus;      // The Thread's state.
```

Comment:
I think this is the way OpenJDK works. We should, if possible, use the same field name and state constants for improved compatibility. Is this the case here?

Reply:
yes